app/src/main/java/com/github/se/travelpouch/ui/dashboard/map/ActivitiesMapScreen.kt

```diff
@@ -156,9 +215,198 @@ fun ActivitiesMapScreen(
                 Icon(imageVector = Icons.Filled.ArrowBack, contentDescription = "Go Back")
               }
         }
-      })
+      }
+}
+
+// -------------------------------------------------
+// Helper Functions for Displaying Information in Bottom Sheet
+// -------------------------------------------------
+
+/**
+ * Displays detailed information about an activity, including:
+ * - Title, date, and time.
+ * - Location name and description.
+ *
+ * @param activity The activity to display.
+ */
+@Composable
+fun ActivityDetailsContent(activity: Activity) {
+  // Date formatter
+  val dateFormatter = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
+  val formattedDate = dateFormatter.format(activity.date.toDate())
+
+  // Time formatter
+  val timeFormatter = SimpleDateFormat("HH:mm", Locale.getDefault())
+  val formattedTime = timeFormatter.format(activity.date.toDate())
+
+  Column(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
```

Comment:
I would put a lazy column, with a fixed size to avoid the bottom sheet being too big

Reply:
yes it is a good idea !